### PR TITLE
WIP improving people page design 

### PR DESCRIPTION
**Pull request by Alaboudi1**
new 
https://alaboudi1.github.io/devx.cs.gmu.edu/people/
current
https://devuxd.github.io/devx.cs.gmu.edu/people/ 

Please let me know what you think!

### CODE REVIEW

**Comment by SaharMehrpour** [comment]
Thank you Abdulaziz. It is far better than the current design.
Just a few points.
- The readme is not detailed that much, but maybe it worths being updated. I think you updated a few `role`s in `_members` data which are reasonable, and I think [readme](https://github.com/devuxd/devx.cs.gmu.edu/tree/master/docs_src#_members) can be updated accordingly. 
- What would the design look like if a member doesn't have a picture? How about having a dummy picture for members without ones? E.g., adding a `elsif` [here](https://github.com/devuxd/devx.cs.gmu.edu/pull/12/files#diff-d9b1af1aced289d1da96b0cfcbe50551L144).

File: docs_src/_layouts/people.html [hunk 1]
```diff
@@ -5,17 +5,17 @@
...
-    {% endif %}
+{% if member.role == 'phd_student' %}
+{% assign phd_student_exists = true %}
+{% elsif member.role == 'master_student' %}
+{% assign master_student_exists = true %}
+{% elsif member.role == 'undergrad_student' %}
+{% assign undergrad_student_exists = true %}
+{% elsif member.role == 'highschool_student' %}
+{% assign highschool_student_exists = true %}
+{% elsif member.role == 'Alumnus' %}
```
Comment (luminaxster) [inline docs_src/_layouts/people.html:16]: Alumni?

File: docs_src/_layouts/people.html [hunk 2]
```diff
@@ -27,118 +27,124 @@
...
         </div>
+        {% endif %}
+        {% endfor %}
+    </div>
     {% endif %}
 
 
     {% if REU_exists %}
-        <h1>REU</h1>
+        <h1>Alumnus</h1>
```
Comment (luminaxster) [inline docs_src/_layouts/people.html]: Alumni

**Comment by Alaboudi1** [comment]
Thanks everyone for the feedback. I will be working on adding more people before merging this pull request.

**Comment by luminaxster** [comment]
Quick question, are the fonts in the website consistent? Aziz, don't push changes in the docs/ folder, they will be flushed once the PR is merged.

File: docs_src/_layouts/people.html [hunk 3]
```diff
@@ -5,17 +5,17 @@
...
-    {% endif %}
+{% if member.role == 'phd_student' %}
+{% assign phd_student_exists = true %}
+{% elsif member.role == 'master_student' %}
+{% assign master_student_exists = true %}
+{% elsif member.role == 'undergrad_student' %}
+{% assign undergrad_student_exists = true %}
+{% elsif member.role == 'highschool_student' %}
+{% assign highschool_student_exists = true %}
+{% elsif member.role == 'alumni' %}
```
Comment (luminaxster) [inline docs_src/_layouts/people.html]: Alumni:= gender-neutral and plural, Alumnus:=male gender and singular, Alumna: female gender and singular. =)

**Review by luminaxster — APPROVED** [review]
Tips: Remove the docs folder from the PR next time.
Suggestions: maybe the font at 700 is too bold
Todos: alumni in its section title, alumnus/alumna in the role name.

**Comment by Alaboudi1** [comment]
I see @luminaxster. We are not supposed to push the doc folder because it get generated by the GitHub workflow. I think in this case we may need two .gitignore files. One for local dev and the other for the workflow. I will see if I can fix that also.

File: docs_src/_layouts/people.html (hunk 3 above)
Comment (luminaxster) [inline docs_src/_layouts/people.html]: {% elsif member.role == 'alumna' or member.role == 'alumnus' %}

File: docs_src/_members/Gennie.md [hunk 4]
```diff
@@ -1,14 +1,16 @@
 ---
 layout: member
 key: Gennie
-role: REU
+role: alumni
```
Comment (luminaxster) [inline docs_src/_members/Gennie.md]: role: alumna

**Review by luminaxster — APPROVED** [review]
just format.

**Comment by luminaxster** [comment]
Let me know when you are done with changes, to review and merge =)

**Comment by Alaboudi1** [comment]
@luminaxster Done!

**Pull request merged**